src/Functions/FunctionsAES.h

```diff
@@ -426,6 +427,7 @@ class FunctionDecrypt : public IFunction
     size_t getNumberOfArguments() const override { return 0; }
     ColumnNumbers getArgumentsThatAreAlwaysConstant() const override { return {0}; }
     bool useDefaultImplementationForConstants() const override { return true; }
+    bool useDefaultImplementationForNulls() const override { return false; }
```

Comment:
I don't understand what was the error and why this is needed?

Reply:
Most of the ciphers threat empty ciphertext as an error, so we have to manually distinguish between "empty because empty" and "empty because null" scenarios and just skip decryption in the latter case.

Reply:
It's not clear for me.
The result cannot be empty because of NULL, the function should return NULL.